Flutter/calendar/appointments.md

```diff
@@ -666,7 +666,7 @@ You can add the recurrence exception appointments and recurrence exception dates
 You can delete any of occurrence, which is an exception from the recurrence pattern appointment by using the `recurrenceExceptionDates` property of `Appointment`. The deleted occurrence date will be considered as recurrence exception dates.
 
 {% tabs %}
-{% highlight dart hl_lines="24" %}
+{% highlight dart hl_lines="25" %}
```

Comment:
https://help.syncfusion.com/flutter/calendar/appointments#add-exception-appointment-to-the-recurrence-series

Highlight recurrence exception dates

Reply:
exception date and appointment highlighted.